addons/survey/models/survey_question.py

```diff
@@ -109,7 +109,9 @@ class SurveyQuestion(models.Model):
     # -- display & timing options
     is_time_limited = fields.Boolean("The question is limited in time",
         help="Currently only supported for live sessions.")
-    time_limit = fields.Integer("Time limit (seconds)")
+    is_time_customized = fields.Boolean(
+        "Customized speed rewards", compute='_compute_is_time_customized', readonly=True, store=True)
```

Comment:
Why is this field stored?

Reply:
Because its value should sometimes be updated based on actions done on the survey and not its direct dependencies (see `_update_default_speed_rating`).
When we change the default limit in survey, we recompute if the a customized question is still customized or not.
Ex: If it was 20 s on question and 30 s on the survey, and survey is changed to 20 s, the question should be updated to not customized.
We cannot do this as a dependency on survey's value, because all questions that had the same original value from the survey (30) would be computed as customized when survey is updated to 20, and they would stay at 30.
Therefore, we update that manually when survey values are changed, and store the result.

Reply:
Even, it doesn't need to be stored.
If the field dependencies is
```python
@api.depends('is_time_limited', 'time_limit', 'survey_id.session_speed_rating_time_limit')
```

The ORM will happily invalidate the field when the value changes on the survey.
This field is only used for UI. No search is done on it. Storing it is a waste of time and disk space.

Reply:
> No search is done on it

Yes there is 
https://github.com/odoo/odoo/pull/115141/files#diff-cf2aadbf6a252dfca6d3450a0ce6ad280b1924afe9a8fc22f7077c998d080009R533

As I said, we cannot add it in the dependencies, or when you change `session_speed_rating_time_limit`, questions with the previous values of `session_speed_rating_time_limit` will be marked as customized which is not what we want.

Reply:
When I say "search", I'm talking about usage in domains. This is not a search per se. You just read the value to filter. A `if` is not a search.

Reply:
OK, there is no "search" in your sense then, but we still need to store the field, right?